js/main.js

```diff
@@ -432,19 +433,23 @@ $elmPreviewList.addEventListener('click', editClicked);
 function editingChoices() {
   for (let k = 0; k < data.weatherOptions.length; k++) {
     if (data.template[data.weatherOptions[k]] === true) {
-      $weatherChoicesList.children[k].children[0].children[0].className = 'far fa-check-circle';
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-check-circle', true);
+      $weatherChoicesList.children[k].querySelector('.far').classList.toggle('fa-circle', false);
```

Comment:
Try to avoid doing stuff like this. Here you are querying the child element twice for the same element. Instead query for it once and put it in a variable.

Then once you have the element, you can modify/update it as much as you want without asking the browser to look for it over and over.